.github/workflows/pr-open.yml

```diff
@@ -82,14 +82,15 @@ jobs:
             overwrite: true
             parameters: -p MIN_REPLICAS=1 -p MAX_REPLICAS=2 -p BUILD=snapshot-${{ github.event.number }}
     steps:
-      - uses: bcgov-nr/action-deployer-openshift@v0.1.0
+      - uses: bcgov-nr/action-deployer-openshift@v1.0.0
         with:
           file: ${{ matrix.file }}
           oc_namespace: ${{ secrets.OC_NAMESPACE }}
           oc_server: ${{ secrets.OC_SERVER }}
           oc_token: ${{ secrets.OC_TOKEN }}
           overwrite: ${{ matrix.overwrite }}
           penetration_test: false
+          verification_path: "/check"
```

Comment:
Remove the **/** at the beginning of the path please.

@DerekRoberts this is a small problem with this, as curl fails if we have double **/** in the path. Can you check that please?

Reply:
Removing the starting slash make the value not intuitive I think. Can we check in the bash script if it's there then remove it? @paulushcgcj

Reply:
Understood. I'll revise the action to work explicitly with or without the slash.